### PR TITLE
Add namespace when we describe pod

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

Add namespace when we describe pod

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -423,7 +423,7 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 	}
 
 	metrics.BindingLatency.Observe(metrics.SinceInMicroseconds(bindingStart))
-	sched.config.Recorder.Eventf(assumed, v1.EventTypeNormal, "Scheduled", "Successfully assigned %v to %v", assumed.Name, b.Target.Name)
+	sched.config.Recorder.Eventf(assumed, v1.EventTypeNormal, "Scheduled", "Successfully assigned %v/%v to %v", assumed.Namespace, assumed.Name, b.Target.Name)
```
Comment (CaoShuFeng) [inline pkg/scheduler/scheduler.go]: The Event objet itself is namespaced?

Comment (tanshanshan) [reply pkg/scheduler/scheduler.go]: the event contains its namespaces, but other event also recorded the namspaces in reason, so I did it .

**Comment by bsalamat** [comment]
This change is fine. Please add "NONE" in the release note section.

**Pull request merged**